app/grandchallenge/reader_studies/forms.py

```diff
@@ -345,6 +354,14 @@ def __init__(self, *args, reader_study, **kwargs):
                     f"hx-include='[id=id_answer_type]'>"
                     f"</div>"
                 ),
+                HTML(
+                    f"<div "
+                    f"hx-get={reverse_lazy('reader-studies:question-interactive-algorithms', kwargs={'slug': reader_study.slug})!r} "
```

Comment:
We should be using `format_html` whenever we're templating out HTML, never using f-string substitution.